asdf.md

```diff
@@ -0,0 +1,336 @@
+## Functions
```

Comment:
Whoops, that's the interim file that jsdoc2md generates that I forgot to remove. Deleted.